src/Atmos/Dycore/test/rising_thermal_bubble.jl

```diff
@@ -155,8 +155,11 @@ function main(mpicomm, DFloat, ArrayType, brickrange, nmoist, ntrace, N,
   @printf(io, "||Q||₂ ( final ) =  %.16e\n", engf)
   @printf(io, "||Q||₂ (initial) / ||Q||₂ ( final ) = %+.16e\n", engf / eng0)
   @printf(io, "||Q||₂ ( final ) - ||Q||₂ (initial) = %+.16e\n", eng0 - engf)
```

Comment:
We probably don't want to be printing diagnostics on every test run, so maybe add an option?
Ideally, could we turn these into tests, e.g
```
@test engf ≈ eng0
```

Reply:
> We probably don't want to be printing diagnostics on every test run, so maybe add an option?

I can suppress the output.

> ```
> @test engf ≈ eng0
> ```

The methods we're using do not guarantee this, we will generally lose some energy though we could in principle gain some too

Reply:
Note there is still a lot of chatter on the tests because of the balance law solver. I'll turn this off on the next balance law solver pull request.